io.github.monsterbirdlimited.soscap.yaml

```diff
@@ -0,0 +1,81 @@
+---
+app-id: io.github.monsterbirdlimited.soscap
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: soscap
+finish-args:
+  - --filesystem=home
```

Comment:
why do you need "home" filesystem?

Reply:
it needs to write to /home/deck/videos
is it unnecessary?

Reply:
if you expect `/home/deck` then there is no chance this can work. You already grant `xdg-videos` that should be enough git the standard directory for videos.

Reply:
the steam deck will not allow you to change the name from deck and always expects /home/deck as far as i am aware.

Reply:
I see you already allow `xdg-videos` below. Shouldn't that be enough?